backend/packages/Upgrade/test/integration/ExperimentStats/IndividualEnrollment.ts

```diff
@@ -115,29 +136,7 @@ export default async function testCase(): Promise<void> {
     ])
   );
 
-  experimentConditionAssignments = await getAllExperimentCondition(experimentUsers[0].id, new UpgradeLogger());
```

Comment:
Why remove this code? It is checking that if the user comes back again it won't get any assignment. By removing this code the test case is no longer checking that scenario. You did this in many test cases please fix them

Reply:
Okay, added this.